pages/dkp/konvoy/2.2/image-builder/create-azure-image/index.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: layout.pug
+navigationTitle: Create Azure Image
+title: Create a custom Azure Image
+excerpt: Learn how to build a custom Azure Image for use with Konvoy
+beta: false
+enterprise: false
+menuWeight: 60
+---
+
+This procedure describes how to use the Konvoy Image Builder to create a [Cluster API](https://cluster-api.sigs.k8s.io/) compliant Azure Virtual Machine (VM) Image that contains the base operating system you specify as well as all the necessary Kubernetes components.   The Konvoy Image Builder uses variable [`overrides`][overrides] to specify base image and container images to use in your new Azure VM image.
```

Comment:
```suggestion
This procedure describes how to use the Konvoy Image Builder to create a [Cluster API](https://cluster-api.sigs.k8s.io/) compliant Azure Virtual Machine (VM) Image. The VM Image contains the base operating system you specify and all the necessary Kubernetes components. The Konvoy Image Builder uses variable [`overrides`][overrides] to specify base images and container images to use in your new Azure VM Image.
```

Reply:
Added, with the exception of the `base images`, changed that to `the base image`, since you can only have one base image.